src-frontend/app/controllers/procedure.js

```diff
@@ -63,6 +64,29 @@ export default Ember.ObjectController.extend({
             });
 
             this.endPropertyChanges();
+        },
+        generateProtocol: function() {
+            var filename = this.get('model.title') + '.xml';
+            $.ajax({
+                beforeSend: function(xhr) {
+                    xhr.setRequestHeader('Authorization', 'Token ' + Cookies.get('authorizationToken'));
```

Comment:
I thought @Ommy configured all of our AJAX calls to automatically insert the token? If not, we should have our own custom AJAX caller that automatically sets our default config.